includes/DataObjects/User.php

```diff
@@ -822,8 +825,15 @@ public function oauthCanCreateAccount()
 		}
 	}
 
+	/**
+	 * @return bool
+	 */
 	public function oauthCanCheckUser()
```

Comment:
may be worth making this protected if it's only every called from here and CommunityUser.